src/app/models/video-item.model.ts

```diff
@@ -0,0 +1,44 @@
+export interface IVideoItem {
+  kind: string
+  etag: string
+  id: string
+  snippet: IVideoSnippet
+  statistics: IVideoStatistics
+}
+
+interface IVideoSnippet {
+  publishedAt: string
+  channelId: string
+  title: string
+  description: string
+  thumbnails: {
+    default: IVideoThumbnail
+    medium: IVideoThumbnail
+    high: IVideoThumbnail
+    standard: IVideoThumbnail
+    maxres: IVideoThumbnail
+  }
+  channelTitle: string
+  tags: string[]
+  categoryId: string
+  liveBroadcastContent: 'none' | string
```

Comment:
This union is the same as just `string`

Reply:
but when you'll actually know which values are available for this prop (for example `'none' | 'default' | 'somethingelse'`), go back and replace `string` types to unions (or enums if needed) whereas possible.